src/app/dataValidator/dataValidator.js

```diff
@@ -44,11 +82,11 @@ const validateData = data => {
   log('\nValidation complete!');
 };
 
-// invoke the validator for WIP purposes
 const data = require('../../../data/test/scenario-01.json');
 
 validateData(data);
 
 module.exports.validateNode = validateNode;
+module.exports.validateProperties = validateProperties;
 module.exports.validateBlock = validateBlock;
 module.exports.validateData = validateData;
```

Comment:
Can you please use ES6 syntax here? so `export default validateData;`

Reply:
That is ES Modules syntax and to run this as a node command it needs to be CommonJS. If we were to use ES Modules syntax we would need to transpile this with babel before running as a node command. 

More info: https://stackoverflow.com/questions/40294870/module-exports-vs-export-default-in-node-js-and-es6#answer-40295288